tests/test_backend.py

```diff
@@ -1068,11 +1068,15 @@ def test_sign_int(self):
         self._run_test_case([_OUTPUT], {_INPUT: x_val})
```

Comment:
btw: does test_sign_int need opset9?

Reply:
> btw: does test_sign_int need opset9?

it does not. opset7 and opset8 also work fine with it.

Reply:
> and do we still need test_sign_int since int type is already covered in test_sign

it seems unnecessary now, should I just remove it?

Reply:
then just remove it.